emails/upvote-talks-reminder.md

```diff
@@ -0,0 +1,25 @@
+Hi,
+
+ 
+
+We're happy that you're coming for PyCon India this year. At this moment, we have about < x > proposal submissions, with < y > days in hand before the CFP closes. One of the things that we consider while selecting talks is the number of people interested in attending that talk. There are many more factors that we consider, like past speaking experience of the speaker, depth of the talk, novelty of the subject etc. 
+
+ 
+
+Please let us know the talks that you would like to attend by upvoting them in funnel.
+
+ 
+
+http://in.pycon.org/funnel/<year>/
+
+ 
+
+Pay attention to talks in the workshops sections. They are 3-hour hand-on workshops and participants have to buy a separate ticket if they want to attend one. Please upvote a workshop only if you are planning to attend it.
```

Comment:
`Pay attention to talks in the workshops sections`, would you like to reword this ?